airflow/www/templates/airflow/graph.html

```diff
@@ -169,9 +169,9 @@
     d3.selectAll("g.node").on("click", function(d){
         task = tasks[d];
         if (task.task_type == "SubDagOperator")
-            call_modal(d, execution_date, true);
+            call_modal(d, execution_date, task_instances[d].try_number, true);
         else
-            call_modal(d, execution_date);
+            call_modal(d, execution_date,task_instances[d].try_number);
```

Comment:
Missing space here, auch!

Reply:
Why did the CI not complain 🤔

Reply:
Javascript is not analyzed by the linter.  I am working on a PR that will delete the jinja directives from Javascript code so that the linter can read the files correctly. 
Reference: 
https://github.com/apache/airflow/pull/4787

Other peoples also working on linting: 
https://github.com/apache/airflow/pull/3656

I have a ready patch that integrates the linter with Travis, but is waiting for the right moment.
https://github.com/PolideaInternal/airflow/pull/75/files

I hope my explanations will be helpful to you.